flink-dist/src/main/resources/flink-conf.yaml

```diff
@@ -46,7 +46,7 @@ jobmanager.heap.size: 1024m
 #
 # Note this accounts for all memory usage within the TaskManager process, including JVM metaspace and other overhead.
 
-taskmanager.memory.process.size: 1568m
+taskmanager.memory.process.size: 1728m
```

Comment:
is this fully derived from existing config options? Maybe we could generate parts of the `flink-conf.yaml`

Reply:
I would not say so. Every time it is a custom decision.